router/gossip.go

```diff
@@ -224,6 +228,16 @@ func (c *GossipChannel) GossipBroadcast(update GossipData) error {
 	return c.relayBroadcast(c.ourself.Name, update)
 }
 
+func (c *GossipChannel) LeaderElect() PeerName {
+	highest := PeerName(0)
+	c.ourself.Router.Peers.ForEach(func(peer *Peer) {
+		if highest < peer.Name {
+			highest = peer.Name
+		}
+	})
+	return highest
+}
+
```

Comment:
This looks like it's in the wrong place. There is one call site (outside test code) and that needs a cast.